### ISSUE TITLE
GUI Window is not opening on Ubuntu 20.04

### ISSUE THREAD

**Issue by Shehab7osny** [deprov447/FakeFeed#6]
I followed all the steps required for installation (Including the dependencies) on Ubuntu 20.04, but unfortunately after running the command `sudo bin/FakeFeed` the following messages have been printed to the terminal and the GUI window didn't initiate.
```
QStandardPaths: XDG_RUNTIME_DIR not set, defaulting to '/tmp/runtime-root'
QQmlComponent: Component is not ready
QML engine started
```

**Comment by deprov447** [comment]
`QStandardPaths: XDG_RUNTIME_DIR not set` can be resolved by exporting a suitable directory path. Or just run `sudo XDG_RUNTIME_DIR='<some-relevant-directory-path>' bin/FakeFeed`

And regarding `QQmlComponent: Component is not ready`, it isn't clear why it is happening. But If QML engine is initiated, I guess it _should_ start. If there is any other thing logging to console, please share.

;Check once if process is running (`ps aux | grep FakeFeed`) and also make sure to wait long enough for GUI to start

**Comment by deprov447** [comment]
@Shehab7osny any follow ups?

**Comment by Shehab7osny** [comment]
@deprov447 Nope! I tried out various approaches but unfortunately none of them worked.
